frameworks/Scala/finagle/build.sbt

```diff
@@ -1,4 +1,4 @@
-lazy val finagleVersion = "18.9.1"
+lazy val finagleVersion = "18.10.0"
 
 name := "finagle-benchmark"
 scalaVersion := "2.12.5"
```

Comment:
should we update scala versions? we're on [2.12.7](https://github.com/twitter/finagle/blob/develop/build.sbt#L58)